Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -92,17 +95,15 @@ extension Message {
     try jsonUTF8Data.withUnsafeBytes { (body: UnsafeRawBufferPointer) in
       if body.count > 0 {
```

Comment:
In rereading this, I'm wondering if `guard...else` to have _early outs_ instead of all the nesting might also be a better solution.

Reply:
Good point.  While making this change, I noticed that the guard here was being handled differently for `init(jsonString:)`, which dominoed into some other related changes...